### PR TITLE
Update palindrom-client and add pal-error-catcher

### PR DESCRIPTION
**Pull request by alshakero**
Follow up to https://github.com/Palindrom/palindrom-client/issues/69

### CODE REVIEW

File: src/StarcounterClientFiles/wwwroot/sys/palindrom-error-catcher/wct.conf.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (tomalec) [inline src/StarcounterClientFiles/wwwroot/sys/palindrom-error-catcher/wct.conf.json:1]: For the next releases, we could add `wct.conf.json` and `Gruntfile.js` to bower ignores

**Review by tomalec — APPROVED** [review]
LGTM

**Comment by PatrykSzwer** [comment]
Update bower-list please.

**Review by PatrykSzwer — APPROVED** [review]
Now it LGTM :)

**Pull request merged**